### ISSUE TITLE
Cannot connect

### ISSUE THREAD

**Issue by DavidLegard** [rundis/elm-light#6]
I don't know if it's a permissions issue, or a path issue, but I get this failure:

```
bootstrap.js[17085]:    
hello world
bootstrap.js[17085]:    
:TEST

Error: read ECONNRESET
    at exports._errnoException (util.js:837:11)
    at TCP.onread (net.js:544:26)
```

Recently, I started getting an extra error

```
Failed to load resource: net::ERR_CONNECTION_TIMED_OUT: http://app.kodowa.com/_fetch
```

**Comment by rundis** [comment]
A couple of questions:
a) What os ?
b) How did you install elm (npm ? platform installer), and are you sure the elm executables are in your path when starting Light Table (and how do you start Light Table) ?
c) What steps did you take before the error occured ?
d) The "hello world" and :TEST seems unrelated to the problem at hand ?
e) What version of Light Table ?

The last error is not related I think (unless it's something you got when displaying the elm package manager screen). When did you get it ? Can you reproduce it or atleast shed some light (no pun intended) when the last error occurs.

**Comment by DavidLegard** [comment]
a) What os ?
- Windows 8, then Windows 10

b) How did you install elm (npm ? platform installer), and are you sure the elm executables are in your path when starting Light Table (and how do you start Light Table) ?
- Elm Windows Installer
- Executables are in path as I can 'elm make' etc from the console
- Double-click desktop icon

c) What steps did you take before the error occured ?
- View -> Connections -> Add Connection -> Elm -> (select root folder)

d) The "hello world" and :TEST seems unrelated to the problem at hand ?
- Yes, I guess so, I just didn't know where those lines came from

e) What version of Light Table ?
- 0.7.2

On 11/12/2015 19:10:47, Magnus Rundberget notifications@github.com wrote:
A couple of questions:
a) What os ?
b) How did you install elm (npm ? platform installer), and are you sure the elm executables are in your path when starting Light Table (and how do you start Light Table) ?
c) What steps did you take before the error occured ?
d) The "hello world" and :TEST seems unrelated to the problem at hand ?
e) What version of Light Table ?
The last error is not related I think (unless it's something you got when displaying the elm package manager screen). When did you get it ? Can you reproduce it or atleast shed some light (no pun intended) when the last error occurs.

**Comment by rundis** [comment]
Please have a look at the readme. LightTable 0.8-alpha or later is required (uses node api from electron that isn't present in 0.7.2). Also the readme states that windows (most likely) wont work. You might get some things to work once you upgrade Light Table, but then again i don't know because I haven't had time to test windows. Feel free to add an issue on supporting windows, probably it shouldn't be too hard.

Tx for trying the plugin out, hopefully I can make usable for you too (soonish).

**Comment by DavidLegard** [comment]
Sorry, I was wrong, I am using 0.8-alpha, it was just that the changelog showed 0.7.2.

And I apologise for not looking closely enough at the ReadMe. I get the syntax highlighting, anyway, which is enough for my modest needs.

Thanks for your work on the plugin, it looks like a great addition to the Elm world.

On 11/12/2015 23:23:27, Magnus Rundberget notifications@github.com wrote:
Please have a look at the readme. LightTable 0.8-alpha or later is required (uses node api from electron that isn't present in 0.7.2). Also the readme states that windows (most likely) wont work. You might get some things to work once you upgrade Light Table, but then again i don't know because I haven't had time to test windows. Feel free to add an issue on supporting windows, probably it shouldn't be too hard.
Tx for trying the plugin out, hopefully I can make usable for you too (soonish).

**Comment by rundis** [comment]
@DavidLegard I just released 0.3.0 of he plugin which now should work pretty well with windows (tested with windows 10) Let me know if you have any issues !
